ipa-core/src/ff/galois_field.rs

```diff
@@ -139,6 +139,7 @@ fn clmul<GF: GaloisField>(a: GF, b: GF) -> u128 {
 /// is that this type is not `Send`.
 ///
 /// [`BitValIter`]: bitvec::slice::BitValIter
+#[allow(dead_code)]
```

Comment:
is it used anywhere? if not, would be better to delete this code

Reply:
I agree. I was wondering whether Andy plans to use it in the future. Otherwise, I would have deleted it. He made some changes end of January. @andyleiserson is it ok to delete or do you still intend to use it?

Reply:
I think it should have been removed with #1115. Not sure why the miri check didn't fail on that PR though.

Reply:
@akoshelev @andyleiserson Ok, I have removed it.